src/components/Review/ReviewSectionRowAssigned.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Grid } from 'semantic-ui-react'
+import strings from '../../utils/constants'
+import { ReviewSectionComponentProps } from '../../utils/types'
+
+const ReviewSectionRowAssigned: React.FC<ReviewSectionComponentProps> = ({
+  isAssignedToCurrentUser,
+  assignment,
+}) => {
+  if (isAssignedToCurrentUser) return <Grid.Column>{strings.ASSIGNED}</Grid.Column>
+  const { lastName, firstName } = assignment.reviewer
+  return <Grid.Column>{`${strings.ASSIGNED_TO} ${firstName} ${lastName}`}</Grid.Column>
```

Comment:
Cool, I was wondering where this would be. So will remove comments to revert changes to keep #175 assigned now